app/models/tasks/copy_media_task.rb

```diff
@@ -4,9 +4,36 @@ def start!
     self.options = task_options
     job = fixer_copy_file(options)
     self.job_id = job[:job][:id]
+
+    # send_rexif_job
```

Comment:
not doing the media files too? or planning to expand this PR?

Reply:
Just starting small. Assuming nothing blows up with the image tasks this should get enabled in short order.